docker/builder.ci.Dockerfile

```diff
@@ -0,0 +1,45 @@
+# syntax=docker/dockerfile:1
+
+# https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/
+# https://www.docker.com/blog/cross-compiling-rust-code-for-multiple-architectures/
+# https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/
+# https://github.com/cross-rs/cross/wiki/Recipes#openssl
+
+FROM --platform=$BUILDPLATFORM ubuntu:latest as builder
+
+ARG TARGETARCH
+
+RUN apt-get update && \
+  apt-get install -y build-essential clang cmake protobuf-compiler curl \
+  g++-aarch64-linux-gnu libc6-dev-arm64-cross \
+  openssl libssl-dev pkg-config \
+  && \
+  rm -rf /var/lib/apt/lists/*
+
+# Get Rust
+RUN curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain stable -y
+ENV PATH="/root/.cargo/bin:${PATH}"
+
+RUN rustup target add aarch64-unknown-linux-gnu
```

Comment:
It can, it will just be more ugly, like the next env var would be present but make no sense if we don't install the libraries, but there is no nice way of not adding it AFAIK.

Reply:
Made these conditional, the env var stayed as it was.